ttnn/cpp/ttnn/types.hpp

```diff
@@ -55,6 +58,11 @@ static std::ostream& operator<<(std::ostream& os, const CoreGrid& core_grid) {
     return os;
 }
 
+using tt::tt_metal::GlobalSemaphore;
+using tt::tt_metal::SubDevice;
+using tt::tt_metal::SubDeviceManagerId;
+using tt::tt_metal::v1::experimental::GlobalCircularBuffer;
```

Comment:
Is this necessary? Granted we don't have a great separation between `tt::tt_metal` and `ttnn`, things get intermixed, but it might makes sense to avoid for new code?

@ayerofieiev-tt